source/internal/_filterMap.js

```diff
@@ -0,0 +1,10 @@
+export default function _filterMap(fn, map) {
+  var result = new Map();
+
+  map.forEach((value, key) => {
+    if (fn(value)) {
+      result.set(key, value);
+    }
+  });
+  return result;
+}
```

Comment:
Sorry noticed this after the merge. For performance reasons, we shouldn't use `.forEach()`. See: https://github.com/ramda/ramda/wiki/Ramda-Conventions#use-imperative-constructs-to-improve-performance

CC: @kedashoe

Reply:
@kedashoe @davetapley ARG! I did the thing where you make comments but forget to submit them

My suggest above this one solves this one. I'll through it up in a separate MR